src/helpers/handleError.ts

```diff
@@ -24,6 +24,13 @@ export function parseError(error: unknown, defaultMessage = ErrorList.unknown) {
     const revertMessage = parseRevertReason(message)
     displayedError = gSNMessage || revertMessage || message
   }
+  if (
+    displayedError?.includes(
+      'can\'t access property "error", res2 is undefined'
+    )
+  )
+    displayedError =
+      'Relay server is not available due to a large fee cost of transaction (insufficient funds for gas * price + value). Try again later.'
```

Comment:
`Please, try again later` sounds more gentle for me :)

Reply:
Did it